### DISCUSSION TITLE
I'm always seeing `Failed to trash file` macOS Sonoma

### DISCUSSION THREAD

**Discussion by danillos** [kamiyaa/joshuto#522] [Q&A]
How can I fix it or debug it?

**Comment by ytkimirti** [comment]
Same here

**Comment by PirxTion** [comment]
I've encountered the same issue. Have you figured out how to solve it yet?

**Reply by danillos** [reply]
Not yet

**Reply by lumpsoid** [reply]
@danillos @ytkimirti @PirxTion 

you missed [config file](https://github.com/kamiyaa/joshuto/blob/main/config/joshuto.toml) use_trash option for [delete_files command](https://github.com/kamiyaa/joshuto/blob/main/docs/configuration/keymap.toml.md#delete_files-delete-selected-files-or-current-file-if-none-were-selected)

or you can install something like - [trashy](https://github.com/oberblastmeister/trashy)
to be able to move files to trash bin first, and then clear it separately
